mission_control/navigator_launch/launch/sim2d.launch

```diff
@@ -0,0 +1,25 @@
+<launch>
+    <!-- Run 2D Simulator -->
+    <include file="$(find navigator_2dsim)/launch/owl.launch" />
+
+    <param name="/is_simulation" type="bool" value="True" />
+
+    <!-- Launch Fake Shooter Service Provider -->
+    <node pkg="navigator_shooter" type="ShooterControl.py" name="shooter_controller">
```

Comment:
I copied this in from other launch files, is it strictly necessary here? Also aren't we going to have to rewrite this with Kip's new launcher?

Reply:
If this is nothing then its good to merge.

Reply:
Good catch, I don't think anything uses that functionality so I'll remove it from the launch files in sim and IRL.